apps/job-service/src/jobs/delayed-jobs.controller.ts

```diff
@@ -13,9 +13,9 @@ import { ApiOperation } from "@nestjs/swagger";
 import { Op } from "sequelize";
 import { ExceptionResponse, JsonApiResponse } from "@terramatch-microservices/common/decorators";
 import { buildJsonApi, getDtoType, JsonApiDocument } from "@terramatch-microservices/common/util";
-import { DelayedJobDto } from "./dto/delayed-job.dto";
 import { DelayedJob } from "@terramatch-microservices/database/entities";
 import { DelayedJobBulkUpdateBodyDto } from "./dto/delayed-job-update.dto";
+import { DelayedJobDto } from "./dto/delayed-job.dto";
```

Comment:
This should be updated to pull from common, and the instance in this service should be removed.